autoload/maktaba/syscall.vim

```diff
@@ -296,9 +267,16 @@ endfunction
 ""
 " @dict Syscall
 " Executes the system call asynchronously and invokes {callback} on completion.
-" {callback} function will be called with the following arguments:
-" {callback}(result_dict), where result_dict contains stdout, stderr and status
-" (code).
+" If vim implementation does not support async execution (details below),
```

Comment:
Perhaps "If the current Vim configuration does not...", since it's partly down to things like `--servername` that aren't part of the implementation per se?